src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBacklogPage.kt

```diff
@@ -15,8 +16,8 @@ class ViewBacklogPage(
     )
 
     fun getIssueKeys(): List<String> {
-        return driver
-            .findElements(By.className("js-issue"))
-            .map { it.getAttribute("data-issue-key") }
+        return JavaScriptUtils.executeScript(driver,
```

Comment:
Nice one. Alternatively we could have used:
```kotlin
driver
    .findElements(...)
    .take(10)
    .map { ... }
```
It'd convert the 1+n into 1+10.
But it's fine to convert it to a 1 (although heavier), if it's less overhead this way.
Is there an easy way of checking whether the 1+10 would cost more or less?